modules/extensions-xml-support/src/test/resources/module-custom-types/module-custom-types-catalog.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'll change the one used in module-param-custom-types-catalog.xml, as using the same file will too much of a trouble for the parameterized test class.